src/scripts/main.js

```diff
@@ -1 +1,60 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve(`First promise was resolved`);
+  });
+
+  setTimeout(() => {
+    reject(new Error(`First promise was rejected`));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('auxclick', (evt) => {
+    if (evt.button === 2) {
+      resolve(`Second promise was resolved`);
+    }
+  });
+
+  document.addEventListener('click', (evt) => {
+    resolve(`Second promise was resolved`);
+  });
+});
```

Comment:
The second promise resolves on both right-click and left-click events. This means it will resolve twice if both events occur. If this is not the intended behavior, you may need to adjust the logic.